presto-main/src/main/java/com/facebook/presto/sql/planner/iterative/rule/ImplementFilteredAggregations.java

```diff
@@ -87,6 +88,9 @@ public Result apply(AggregationNode aggregation, Captures captures, Context cont
             if (call.getFilter().isPresent()) {
                 Expression filter = call.getFilter().get();
                 Symbol symbol = context.getSymbolAllocator().newSymbol(filter, BOOLEAN);
+                if (mask.isPresent()) {
```

Comment:
remove, dead code.

Reply:
If this is dead, the maybe `verify(!mask.isPresent())`, especially that we create new mask 2 lines  below.